src/api/app/views/webui2/webui/attribute/index.html.haml

```diff
@@ -0,0 +1,48 @@
+- @pagetitle = "Attributes of #{@package} (Project #{@project})"
+- package_bread_crumb 'Attributes'
+
+.card
+  = render(partial: 'webui/package/tabs', locals: { project: @project, package: @package })
+
+  .card-body
+    %h3= @pagetitle
+    - if @attributes.present?
+      .table-responsive
+        %table.table.table-hover#attributes
+          %thead.thead-light
+            %tr
+              %th.col-auto{ scope: 'col' } Attributes
+              %th.col{ scope: 'col' } Values
+              - if policy(@package).update?
+                %th.col-auto{ scope: 'col' } Actions
+          %tbody
+            - @attributes.each do |attribute|
+              %tr
+                %td= attribute.fullname
+                %td
+                  %ul
+                    - unless attribute.values.empty?
+                      - attribute.values.each do |v|
+                        %li.value
+                          %pre.d-inline= v.value
```

Comment:
That's weird... `value.value` :unamused:

Reply:
I cannot do much here... it would involve some refactoring, which is outside the scope of this PR.